tests/e2e/test_driver.py

```diff
@@ -633,6 +635,27 @@ def test_closing_a_closed_connection_doesnt_fail(self):
             
             self.assertTrue(expected_message_was_found, "Did not find expected log messages")
 
+    @skipUnless(pysql_supports_arrow(), 'needs arrow support')
+    def test_cloud_fetch(self):
```

Comment:
Since this is a long running test, should we put it into the large query suite?

Reply:
That's a great suggestion, I moved it there. I was also worried about how this test may take a long time compared to the other core tests in the suite